### DISCUSSION TITLE
Wrong IntelliSense import path on VSCode

### DISCUSSION THREAD

**Discussion by giopetris** [remix-run/remix#608] [Q&A]
Whenever I use the Intellisense on VSCode to import something from `remix` package it shows `@remix-run/server-runtime` instead. Does anyone know how I can fix that?

![image](https://user-images.githubusercontent.com/1332141/143308321-7d88b166-e4a1-482c-8c81-98db2aab966c.png)

**Comment by kof** [comment]
Duplicate question here https://github.com/remix-run/remix/discussions/1591
